src/main/java/org/broadinstitute/consent/http/service/DraftService.java

```diff
@@ -0,0 +1,174 @@
+package org.broadinstitute.consent.http.service;
+
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import jakarta.ws.rs.BadRequestException;
+import jakarta.ws.rs.NotAuthorizedException;
+import jakarta.ws.rs.NotFoundException;
+import jakarta.ws.rs.core.StreamingOutput;
+import java.io.InputStream;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.broadinstitute.consent.http.db.DraftDAO;
+import org.broadinstitute.consent.http.enumeration.UserRoles;
+import org.broadinstitute.consent.http.models.DraftInterface;
+import org.broadinstitute.consent.http.models.DraftSummary;
+import org.broadinstitute.consent.http.models.FileStorageObject;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.util.gson.GsonUtil;
+import org.glassfish.jersey.media.multipart.FormDataBodyPart;
+import org.jdbi.v3.core.Jdbi;
+
+public class DraftService {
+
+  private final Jdbi jdbi;
+  private final DraftDAO draftDAO;
+  private final DraftFileStorageService draftFileStorageService;
+
+  @Inject
+  public DraftService(Jdbi jdbi, DraftDAO draftDAO,
+      DraftFileStorageService draftFileStorageService) {
+    this.jdbi = jdbi;
+    this.draftDAO = draftDAO;
+    this.draftFileStorageService = draftFileStorageService;
+  }
+
+  public void insertDraft(DraftInterface draft)
+      throws SQLException, BadRequestException {
+    jdbi.useHandle(handle -> {
```

Comment:
why have the jdbi.useHandle logic here in the service instead of in the DAO? It looks like it lives in the DAO for most other cases